Projects quality of life

aryand10 opened the pull request:
Add character count limit for task title to better adhere to restrictions set by calendar page

Commit:
Update JSDoc documentation

aryand10 committed:
Setup character count limit for task title

Character count limit for task title set up for 50 characters

aryand10 committed:
Merge branch 'projects-quality-of-life' of https://github.com/cse110-sp24-group11/cse110-sp24-group11 into projects-quality-of-life

Commit:
Update JSDoc documentation

ezh247467 reviewed (changes requested):
Sorry @aryand10 I had added a couple things to the format of how I'd display the Project tasks for the Calendar since we last talked, so I'm actually going to need you to change the character limit again. A total of 45 is the new limit, but due to including the project title when being displayed, I would also need you to limit the Project title character limit as well. I'm thinking of Project title limit to 20 chars and task title limit to 25 chars to fit the total of 45, but any other combination is fine at your discretion as long as the total does not exceed 45. Apologies for not letting you know sooner.

aryand10 committed:
Restricted Project Title and Task Title Length

Based on review from @ezh247467 project title is set to max length of 20 and task title is set to max length of 45

Commit:
Update JSDoc documentation

ezh247467 reviewed (approved):
Character limit restrictions look good to me!
